### DISCUSSION TITLE
Graphql

### DISCUSSION THREAD

**Discussion by atamersignum** [Budibase/budibase#4184] [Help] [answered]
Hi,
Does Budibase support qraphql?
Thank you

**Comment by shogunpurple** [comment, marked as answer]
Hi @atamersignum,

Not currently. You can find more information about it here. We will ship it at some point over the next few months, but I can't give you a timeline unfortunately.

https://github.com/Budibase/budibase/discussions/1402

**Reply by Yazakk** [reply]
@shogunpurple I really do hope we get GraphQL soon. I really like your service and it's amazing.

**Comment by luke-ransfield** [comment]
+1 for being able to send queries in graphql (I see it's on the road map for August/September?)
